src/NUnitFramework/framework/Interfaces/ICommandWrapper.cs

```diff
@@ -63,4 +63,13 @@ public interface IWrapTestMethod : ICommandWrapper
     public interface IWrapSetUpTearDown : ICommandWrapper
     {
     }
+
+    /// <summary>
+    /// Objects implementing this interface are used to wrap
+    /// tests that can repeat. The implemeting object runs
+    /// each time before the test is repeats.
```

Comment:
If I understood properly, an object implementing `IRepeatTest` is *not* invoked each time that the test is repeated. It's invoked once and may decide to invoke the inner command any number of times. Am I thinking correctly?

Reply:
Answering from memory... the wrapper object (IRepeatTest) is supposed to set up a TestCommand that is added to the chain of commands used in running the test. The command runs once, invoking chained commands as many times as it likes.